crypto/keyring/types.go

```diff
@@ -35,7 +35,7 @@ const (
 	DefaultBIP39Passphrase = ""
 
 	// bits of entropy to draw when creating a mnemonic
-	defaultEntropySize = 256
+	DefaultEntropySize = 256
```

Comment:
To avoid cyclic dep issue, I have updated package for test files inside keyring folder to keyring_test. Therefore, I have to import `keyring.DefaultEntropySize` for testing purposes.